packages/angular_devkit/build_angular/src/webpack/plugins/common-js-usage-warn-plugin.ts

```diff
@@ -10,12 +9,16 @@
 import { isAbsolute } from 'path';
 import { Compiler, compilation } from 'webpack';
 import { addWarning } from '../../utils/webpack-diagnostics';
+import { isWebpackFiveOrHigher } from '../../utils/webpack-version';
 
 // Webpack doesn't export these so the deep imports can potentially break.
 const CommonJsRequireDependency = require('webpack/lib/dependencies/CommonJsRequireDependency');
 const AMDDefineDependency = require('webpack/lib/dependencies/AMDDefineDependency');
 
 // The below is extended because there are not in the typings
+// TODO_WEBPACK_5 type Types of property 'issuer' are incompatible.
+// Type 'WebpackModule | null' is not assignable to type 'Module'
+// @ts-ignore
```

Comment:
Since we don't have Webpack 5 as a dev dependencies we should be able to remove this.

Reply:
this allow's to run unit tests with webpack@5

Reply:
removed